_includes/components/image-example-incorrect.html

```diff
@@ -0,0 +1,31 @@
+{% comment %}
+include
+- image-alt-text
+- image 
+- red-text
+- text
+- fully-bordered-class
```

Comment:
I feel like there must be a way to improve this. I'm passing the same class or nothing.

Reply:
I think we could have this as a boolean attribute of sorts, i.e. `bordered_image=true`, and apply the relevant class based on that?

```
{% if include.bordered_image %}usa-image-example__image--bordered{% endif %}
```

Reply:
Made that [switch](https://github.com/18F/identity-site/pull/1151/commits/5c2871811702fcd440a5d8f2823bbad8dd7dac5b).